src/app/Components/PhoneInput/PhoneInput.tests.tsx

```diff
@@ -114,8 +114,8 @@ describe("PhoneInput", () => {
     expect(tree.root.findAllByType(Text)[2].props.children).toBe("This phone number is incomplete")
   })
 
-  it("does not show a validation message when phone number valid", () => {
-    const tree = renderWithWrappers(
+  it.only("does not show a validation message when phone number valid", () => {
```

Comment:
```suggestion
  it("does not show a validation message when phone number valid", () => {
```
it.only left by mistake?